Agents/MackayCalculatorAgent/api/ModelTwerkHandler.py

```diff
@@ -0,0 +1,23 @@
+# Data Handler to twerk the handler to back to net zero after an specified action on a specified lever, the new plot charts dta are returned
+# This router is still in development and not used for deployment
+# ===============================================================================
+from flask_restful import Api, Resource, reqparse,request
+from mackay.calculator_model import CalculatorModel
+from flask import jsonify
+
+calculator = CalculatorModel()
+class ModelTwerkHandler(Resource):
```

Comment:
Are you sure you are using the word "twerk" correctly? As far as I'm aware it means something very _very_ different. Please google the meaning of it, to be sure that's what you actually mean. Perhaps you mean "tweak"?

Reply:
Nope, that was definitely not what I mean. Yes I mean 'tweak'.